sample-compose/build.gradle.kts

```diff
@@ -102,8 +102,8 @@ koverMerged {
         "com.bumptech.glide.*",
         "dagger.hilt.internal.*",
         "hilt_aggregated_deps.*",
-        "co.nimblehq.template.databinding.*",
-        "co.nimblehq.template.di.*"
+        "co.nimblehq.sample.compose.databinding.*",
+        "co.nimblehq.sample.compose.di.*"
```

Comment:
```suggestion
        "co.nimblehq.template.compose.databinding.*",
        "co.nimblehq.template.compose.di.*"
```

Reply:
Oh, wait, this one is in [sample-compose/build.gradle.kts](https://github.com/nimblehq/android-templates/pull/350/files/99a718f36539580628e76d62a10f31147ced2e71#diff-d38b787b9906da28c79ae1c08c59ed7879dc1842807cb7ec8138cabb22c0a2a3) It seems like it's correctly but Do we need to update this 🤔  @Tuubz As we will have `template-xml/template-compose` So, Do we still need `sample-compose`

Reply:
> Do we need to update this

The update is correct (although it is out of scope, that's OK)

> Do we still need sample-compose

Yes, `sample-compose` is for sample purposes, `template-compose` is used to generate a new project 🚀

Reply:
> Yes, sample-compose is for sample purposes, template-compose is used to generate a new project 🚀

@Tuubz May I know why don't we add the example  of `compose` to `template-compose` instead of `sample-compose` So when developer generates the `template-compose`, then they will see the sample `compose`🤔

Reply:
If we keep the sample code in `template-compose`, then the developer will have to remove all the sample code manually. This is an annoying process, so the purpose of `template-compose` is to start a new project as clean and fast as possible 👍 

Please let me know if I misunderstood your question 🐱

Reply:
> This is an annoying process, so the purpose of template-compose is to start a new project as clean and as fast as possible

@Tuubz What about the `classes` in module `data` and `domain` e.g., `ApiService` , including `unitTest` inside there 🤔

Reply:
The idea of `template` is to provide the minimum amount of code required to get started (quickly), but also highlight the project structure 🚀 

If we remove `ApiService.kt`, then the developer will need to add the setup manually (Ex: `ApiServiceProvider.kt`). It will require more time to add this ourselves, instead of simply renaming it.

For `UseCaseTest.kt`, I think it's acceptable to keep some basic test classes (think about how we generate a new project from Android Studio, they also include a few test classes by default)

Anyways, if you still feel like they should be removed, let's move this discussion to an RFC ✍️